product-details/scss/layout/top-bar.scss

```diff
@@ -0,0 +1,341 @@
+/*------------------------------------*\
+  #TOP BAR
+\*------------------------------------*/
+.top-bar {
+  width: 100vw;
+  background: $white;
+  // color: $white;
+  position: relative;
+  padding: 0.5em 0em;
+
+  .top-bar-contant {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 70%;
+    margin: 0 auto;
+    position: relative;
+    top: 30%;
+
+    .section-logo-lang {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+
+      .logo {
+
+        @include retinaMobile {
+          left: 9%;
+        }
+
+        img {
+          max-width: 8em;
+        }
+      }
+
+      .select-lang {
+
+        &>ul {
+          display: flex;
+          display: inline-block;
+          vertical-align: middle;
+
+          @include mini_laptop {
+            display: inline;
+            color: $secondColorWeprodev;
+            max-width: 64em;
+
+          }
+
+
+          li {
+            list-style: none;
```

Comment:
why we have 2 types of mixin name? retinaMobile is camelCase and  mini_laptop is snake_case ?!